src/collection.ts

```diff
@@ -793,4 +793,19 @@ export default class Collection {
       aggregate: !!options.aggregate,
     });
   }
+
+  /**
+   * Retrieves collection server endpoints.
+   *
+   * @return {Object}
+   */
+  get endpoint(): Record<string, Function> {
+    return {
+      toString: () => endpoint.collection(this.bucket.name, this.name),
+      collection: () => endpoint.collection(this.bucket.name, this.name),
```

Comment:
Same as with Bucket. Let's remove these and use a property directly on `Collection` to represent the URL.